tests/e2e/performance/test_elasticsearch.py

```diff
@@ -0,0 +1,69 @@
+"""
```

Comment:
this file has to be in test/libtest rather than e2e/perf

Reply:
I moved it.